mifospay/src/main/res/values/strings.xml

```diff
@@ -93,7 +93,7 @@
         registered on MifosPay. \n3. It's sim is on this device.
     </string>
     <string name="okay">Okay</string>
-    <string name="trusted_merchants">Trusted \nMerchants</string>
+    <string name="trusted_merchants">Trusted Merchants</string>
```

Comment:
It should be "Trusted **m**erchants", not "Trusted **M**erchants (merchants starting with lower case)